packages/core/src/animation/AnimatorState.ts

```diff
@@ -0,0 +1,149 @@
+import { AnimationClip } from "./AnimationClip";
+import { Entity } from "../Entity";
+import { AnimatorStateTransition } from "./AnimatorTransition";
+import { WrapMode } from "./enums/WrapMode";
+import { PlayType } from "./enums/PlayType";
+import { AnimatorStateType } from "./enums/AnimatorStateType";
+
+export interface AnimatorStateMap {
+  [key: string]: AnimatorState;
+}
+
+export class AnimatorState {
+  /**
+   * The name mapping of all states.
+   */
+  static statesMap: AnimatorStateMap = {};
```

Comment:
Shouldn't static,  different clip in different animator maybe same name usually

Reply:
done.